shared/src/androidMain/kotlin/com/mbta/tid/mbta_app/Helpers.kt

```diff
@@ -14,6 +14,6 @@ internal fun createDataStore(context: Context): DataStore<Preferences> =
 fun initKoin(appVariant: AppVariant, nativeModule: Module, context: Context) {
     startKoin {
         androidContext(context)
-        modules(appModule(appVariant) + platformModule() + nativeModule)
+        modules(platformModule() + appModule(appVariant) + nativeModule)
```

Comment:
Just curious, but why were these swapped?

Reply:
Oh this isn't actually necessary. At first I tried having ErrorBannerStateRepo call `subscribeToNetworkStatusChanges` on init, but that was causing dependency injection issues with `BeanNotFound` errors. I thought maybe the platformModule needed to be called first here since appModule depends on it, but that didn't end up fixing the issue, and I just opted to call `subscribeToNetworkStatusChanges` from the ViewModel instead.